tests/phpunit/tests/SanitizeDate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class SanitizeDate
+ *
+ * @package Traduttore\Tests
+ */
+
+namespace Required\Traduttore_Registry\Tests;
+
+use DateTime;
+use \WP_UnitTestCase;
+use function \Required\Traduttore_Registry\sanitize_date;
+
+/**
+ *  Tests dates by the sanitize_date() function.
+ */
+class SanitizeDate extends WP_UnitTestCase {
+
+	/**
+	 * Ensures good dates are unaltered.
+	 */
+	public function test_good_dates_are_unaltered(): void {
+		$good_dates = [
+			'2020-02-06 09:24:03+0000',
+			'2020-02-06 09:24:03',
+			'2020-02-06',
+		];
+
+		foreach	( $good_dates as $date ) {
+			$this->assertEquals( sanitize_date( $date ), new DateTime( $date ) );
+		}
+	}
+
+	/**
+	 * Ensures timezones are stripped.
+	 */
+	public function test_timezones_are_stripped(): void {
+		$dates_with_timezones = [
+			[
+				'input'    => 'Tue Jan 12 2016 15:04:22 GMT+0100 (Romance Standard Time)',
+				'expected' => 'Tue Jan 12 2016 15:04:22 GMT+0100'
+			],
+			[
+				'input'    => 'Tue Dec 22 2015 12:52:19 GMT+0100 (West-Europa)',
+				'expected' => 'Tue Dec 22 2015 12:52:19 GMT+0100',
+			],
+		];
```

Comment:
We can simplify the test by using a [data provider](https://phpunit.readthedocs.io/en/9.0/writing-tests-for-phpunit.html#writing-tests-for-phpunit-data-providers) for both cases. There should also be a third case for an invalid date to check the return value in case of an exception.

Reply:
Thanks, @ocean90. I switched to a data provider and added a test for the return value for unparseable date strings.